kolibri/core/assets/src/views/core-modal/index.vue

```diff
@@ -224,4 +195,10 @@
   .fade-enter, .fade-leave-active
     opacity: 0
 
+  >>>.buttons
```

Comment:
let's rename `.buttons` something more easily searchable since it's a special class applied from the parent. Maybe something scary and explicit like `.global-core-modal-buttons`?